docs/elements.md

```diff
@@ -190,6 +190,30 @@ Two grading methods are available when using `partial-credit="true"`:
 
 -----
 
+## `pl-dropdown` element
+
+Select the correct answer from a drop-down **select** menu list of potential answers. The potential options are listed as a JSON array.
+
+#### Sample Element
+
+![](elements/pl-dropdown.png)
+
+**question.html**
+
+```html
+<pl-dropdown sort="ascend" options='wise, clumsy, wreckless' answer="wise" answer-key="hume"></pl-dropdown>
```

Comment:
`options` seems to go against the trend of using `<pl-answer>`, though I can see why that would be the case as this element is meant for being inline with the text. 

But, I think `answer-key`  should be given as `correct-answer`.

With this being said, what are your thoughts on something more like:
 
```html
<pl-dropdown sort="ascend" correct-answer="hume">
<pl-answer correct='true'>wise</pl-answer>
<pl-answer> clumsy </pl-answer>
<pl-answer> wreckless </pl-answer>
</pl-dropdown>
```

Reply:
Yeah, the inline formatting was certainly the key consideration. With these sorts of situations, I do not mind going either way if someone wants to make an executive decision on this.

Both are valid approaches. The name change is fine with me too.

Reply:
I am going to start implementing this just to bypass any communication wait :) 

Feedback appreciated!

Reply:
Logic changed to use pl-answer in 8767fb10ad10a3d4f65f3e92fefba8c32bb514fe,  ce8eff47b9056205996d83f12eed6b2e91db52a6, and 948156f6d2ba84620340737e7534a09154e71717.